File: SDKs/gambitcomm.local/mimic/21.00/go/pkg/models/operations/protocoldhcpgettrace.go
package operations

import (
	"openapi/pkg/models/shared"
)

type ProtocolDhcpGetTracePathParams struct {
	AgentNum int32 `pathParam:"style=simple,explode=false,name=agentNum"`
}

type ProtocolDhcpGetTraceRequest struct {
	PathParams ProtocolDhcpGetTracePathParams
}

type ProtocolDhcpGetTraceResponse struct {
	ConfigDHCP  *shared.ConfigDhcp
	ContentType string
	StatusCode  int64
}

File: SDKs/gambitcomm.local/mimic/21.00/python/src/sdk/models/operations/protocol_dhcp_get_trace.py
import dataclasses
from typing import Optional
from ..shared import configdhcp as shared_configdhcp


@dataclasses.dataclass
class ProtocolDhcpGetTracePathParams:
    agent_num: int = dataclasses.field(metadata={'path_param': { 'field_name': 'agentNum', 'style': 'simple', 'explode': False }})
    

@dataclasses.dataclass
class ProtocolDhcpGetTraceRequest:
    path_params: ProtocolDhcpGetTracePathParams = dataclasses.field()
    

@dataclasses.dataclass
class ProtocolDhcpGetTraceResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    config_dhcp: Optional[shared_configdhcp.ConfigDhcp] = dataclasses.field(default=None)
    
